.github/workflows/multi-approvers.yml

```diff
@@ -21,7 +21,16 @@ on:
       # the full path that should work with the https://raw.githubusercontent.com/ prefix.
       # See the README for more.
       org-members-path:
-        required: true
+        description: 'DEPRECATED: Use members-url with https://raw.githubusercontent.com/ prepeneded instead.'
+        type: 'string'
+      members-url:
+        description: 'Full URL to JSON members file. URL must be publicly accessible.'
+        type: 'string'
+      local-members-path:
+        description: 'Path to JSON members file. See README for more.'
+        type: 'string'
+      team:
+        description: 'The GitHub org and team slug that should be used for determining external vs. internal users. Format is ${org}/${team-slug}.'
```

Comment:
I feel like we really need to simplify these inputs. Why do we need so many?

Reply:
I think we need at least two to support GH teams and a members JSON file.

1. GH team descriptor
2. JSON URL

For the JSON URL, it's nice to allow a path to the local repo for manual testing / development / maintenance. This isn't required, but it alleviates the need to find a public place to put a JSON file when testing in a private repo like abcxyz/actions-testsing.

Also note, the JSON members files is nice because it does not require a github-token-minter (Minty) setup.

Reply:
Why do we need to allow specifying a members JSON at all? That seems useful for local testing, but should never be used in a real scenario. We would never actually want someone manually managing the list of internal members in the organization, since that represents a substantial risk of being out of sync and creating insider risk.

I am **strongly** opposed to complicating a public API surface just for testing. These kinds of things should be unit tests anywhere (where we fake the response from the GitHub API).